### ISSUE TITLE
Expand on literature using/citing cclib

### ISSUE THREAD

**Issue by langner** [cclib/cclib#610]
(this is moved from cclib/cclib.github.io#19)

This could go in the docs somewhere.

**Comment by Chokerino** [comment]
Please expand on the issue.

**Comment by langner** [comment]
Well, there are over 2000 citations of the original cclib paper out there: https://scholar.google.com/scholar?oi=bibs&hl=en&cites=18169031345422383864

The idea would be to somehow integrate that in the docs. Don't have a clear idea right now how, though.

**Comment by GomaruSai** [comment]
Hi!
I would like to work on this issue.
My idea is to use a google scholar api to extract the titles of the papers that cite the cclib paper from the scholar page and print it to the reStructuredText file that contains the "About cclib" section in the documentation.
Can you please clarify if this is what is to be done?

**Comment by langner** [comment]
> Hi!
> I would like to work on this issue.
> My idea is to use a google scholar api to extract the titles of the papers that cite the cclib paper from the scholar page and print it to the reStructuredText file that contains the "About cclib" section in the documentation.
> Can you please clarify if this is what is to be done?

I don't think this is useful. What is the advantage of copying that list into our docs?
